### DISCUSSION TITLE
PIN

### DISCUSSION THREAD

**Discussion by mbhforum** [pyalarmdotcom/alarmdotcom#278] [Q&A]
Is there a way in HA to avoid having to put your PIN in to disarm the system?  My Homebridge integration does this without issue.

**Comment by elahd** [comment]
Just keep the PIN setting in the integration blank. (Use CLEAR! to clear
the PIN if you have one set.) That PIN is to set a PIN that Home Assistant
enforces. Alarm.com doesn’t check PINs over the API.

**Comment by mbhforum** [comment]
Thanks!  I am running 3.0.0 Beta 2 and it doesn't appear it's able to set the alarm state at all.  I see this error in the logs.   It's not reading the correct alarm state either.  

![image](https://github.com/pyalarmdotcom/alarmdotcom/assets/11696289/07fa1afa-56fd-4757-af79-daa65509be5a)


Logger: homeassistant.helpers.service
Source: helpers/service.py:757
First occurred: 10:01:52 AM (10 occurrences)
Last logged: 10:04:34 AM

Unable to find referenced entities alarm_control_panel.system or it is/they are currently not available

**Comment by elahd** [comment]
I’m going to release beta 3 today or tomorrow. I’ll work a fix for this
into that release.
